src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(5);
+        String color = colorSupplier.getRandomColor();
+
+        switch (figureType) {
+            case 0:
+                return new Square(color, random.nextDouble() * 10);
+            case 1:
+                return new Rectangle(color, random.nextDouble() * 10, random.nextDouble() * 10);
+            case 2:
+                return new RightTriangle(color, random.nextDouble() * 10, random.nextDouble() * 10);
+            case 3:
+                return new Circle(color, random.nextDouble() * 10);
+            case 4:
+                return new IsoscelesTrapezoid(color, random.nextDouble()
+                        * 10, random.nextDouble() * 10, random.nextDouble() * 10);
+            default:
+                return getDefaultFigure();
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle("white", 10);
+    }
+}
```

Comment:
The class `FigureSupplier` should not have an empty line at the beginning of its definition.